### PR TITLE
Remove project-specific hardcoded configuration

### PR DESCRIPTION
**Pull request by ninaburg**
## Purpose:
This pull request abstracts the project-specific files necessary for running the application on the European Weather Cloud (EWC) and automating execution with Aviso. These files have been migrated to a new repository [MeteoSwiss/flex-container-orchestrator](https://github.com/MeteoSwiss/flex-container-orchestrator) dedicated to the orchestration of the Flexpart pipeline on the EWC.

### CODE REVIEW

File: flexprep/config/settings.yaml [hunk 1]
```diff
@@ -10,11 +10,11 @@ main:
   db_path: /src/db/sqlite3-db
   s3_buckets:
     input:
-      endpoint_url: https://object-store.os-api.cci1.ecmwf.int
-      name: flexpart-input
+      endpoint_url: "${INPUT_FLEXPREP_S3_ENDPOINT_URL}"
```
Comment (cfkanesan) [inline flexprep/config/settings.yaml]: If possible, I would rename this variable to `FLEXPREP_INPUT_S3_ENDPOINT_URL`. Same for the other variables.

**Review by cfkanesan — APPROVED** [review]

**Review by victoria-cherkas — APPROVED** [review]

File: flexprep/config/settings.yaml
Comment (victoria-cherkas) [inline flexprep/config/settings.yaml:1]: Just to say these default settings can be overwritten with environment variables anyway if using mch_python_commons to parse the config, so there is no need to put variables in the config

Comment (victoria-cherkas) [reply flexprep/config/settings.yaml:1]: See https://service.meteoswiss.ch/documentation/mch-python-commons/latest/python/usage.html#environmental-variables-for-settings

File: flexprep/config/settings.yaml (hunk 1 above)
Comment (ninaburg) [reply flexprep/config/settings.yaml]: Okay, with the service context before.  Now the env. var. names are removed from this file now that I know I can overwrite the settings anyway with mch_python_commons.

File: flexprep/config/settings.yaml
Comment (ninaburg) [reply flexprep/config/settings.yaml:1]: Thanks for letting me know that I can overwrite the settings with environment variables with mch_python_commons. I’ve restored the file with the default settings without relying on environment variables. Additionally, I added a prefix in `service_settings.py` to shorten the long environment variable names that point to these settings.

**Pull request merged**